### DISCUSSION TITLE
How to configure the include option in swc-loader

### DISCUSSION THREAD

**Discussion by liu-collab** [web-infra-dev/rsbuild#3373] [Q&A]
![image](https://github.com/user-attachments/assets/ee0c8382-abe1-4d3d-87c6-3e4d649030dc)
在craco中配置了babel-loader include 迁移到rsbuild怎么配置swc-loader

**Comment by liu-collab** [comment]
上面的配置主要是在packages/A  ,  packages/B  在B模块下引用A模块里面的内容

**Comment by chenjiahan** [comment]
Try this: https://rsbuild.dev/config/source/include
